packages/firestore/test/integration/util/helpers.ts

```diff
@@ -27,22 +27,37 @@ declare const __karma__: any;
 
 const PROJECT_CONFIG = require('../../../../../config/project.json');
 
+const EMULATOR_PORT = process.env.FIRESTORE_EMULATOR_PORT;
+const EMULATOR_PROJECT_ID = process.env.FIRESTORE_EMULATOR_PROJECT_ID;
+export const USE_EMULATOR = !!EMULATOR_PORT;
+
+const EMULATOR_FIRESTORE_SETTING = {
+  host: `localhost:${EMULATOR_PORT}`,
+  ssl: false,
+  timestampsInSnapshots: true
+};
+
+const PROD_FIRESTORE_SETTING = {
+  host: 'firestore.googleapis.com',
+  ssl: true,
+  timestampsInSnapshots: true
+};
+
 export const DEFAULT_SETTINGS = getDefaultSettings();
+console.info(`Default Settings: ${JSON.stringify(DEFAULT_SETTINGS)}`);
```

Comment:
We prevent console.log / console.info because: 1) In the SDK, all logging should go through our official logging mechanism, 2) often developers accidentally leave debug console log statements in their PRs and this makes sure they don't get checked in.

So it's okay to use console logging in tests as long as it's used sparingly and it's intentional.

So in this case, I would just add a tslint suppression comment above it:

// tslint:disable-next-line:ban

Reply:
Done.